### DISCUSSION TITLE
How to reproduce the benchmark of GreptimeDB?

### DISCUSSION THREAD

**Discussion by byte-sourcerer** [GreptimeTeam/greptimedb#4150] [Q&A] [answered]
How to reproduce the [benchmark](https://github.com/GreptimeTeam/greptimedb/blob/main/docs/benchmarks/tsbs/v0.8.0.md) of GreptimeDB? Is there a specific script that I can execute directly?

**Comment by waynexia** [comment, marked as answer]
Please check https://github.com/GreptimeTeam/greptimedb/pull/4151
